Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -744,12 +756,29 @@ void NetPlayDialog::OnMsgStopGame()
 
 void NetPlayDialog::OnPadBufferChanged(u32 buffer)
 {
-  QueueOnObject(this, [this, buffer] { m_buffer_size_box->setValue(buffer); });
+  QueueOnObject(this, [this, buffer] {
+    const QSignalBlocker blocker(m_buffer_size_box);
+    m_buffer_size_box->setValue(buffer);
+  });
   DisplayMessage(tr("Buffer size changed to %1").arg(buffer), "");
 
   m_buffer_size = static_cast<int>(buffer);
 }
 
+void NetPlayDialog::OnHostInputAuthorityChanged(bool enabled)
+{
+  QueueOnObject(this, [this, enabled] {
+    bool is_hosting = Settings::Instance().GetNetPlayServer() != nullptr;
```

Comment:
Isn't this able to just be:

```cpp
const bool is_hosting = IsHosting();
```

Since `NetPlayDialog` has an `IsHosting()` function that does the same thing as this line?

Reply:
Thanks, forgot about that function. I can actually just drop the local variable entirely in this case.